saucelabs.karma.conf.js

```diff
@@ -0,0 +1,87 @@
+/**
+ *
+ */
+var webpack = require('webpack');
+var webpackCommon = require('./webpack.common.config.js');
+
+// A subset of the app webpack config
+var webpackTestConfig = {
+    devtool: 'inline-source-map',
+    plugins: [
+        new webpack.ResolverPlugin(
+            new webpack.ResolverPlugin.DirectoryDescriptionFilePlugin('bower.json', ['main'])
+        ),
+
+        // Make sure that CommonJS is always used
+        new webpack.DefinePlugin({
+            'define.amd': false
+        }),
+
+        new webpack.ProvidePlugin({
+            $: 'jquery',
+            jQuery: 'jquery',
+            'window.jQuery': 'jquery',
+            'window.$': 'jquery'
+        }),
+    ],
+    resolve: webpackCommon.resolve,
+    externals: {'jquery': 'jQuery', 'jquery-ui': 'jQuery.ui'},
+    module: {
+        loaders: webpackCommon.module.loaders.concat([
+            // Assume test files are ES6
+            {test: /\.test\.js$/, loader: 'babel-loader'},
+        ])
+    }
+};
+
+var browsers = {
+  sl_chrome: {
+    base: 'SauceLabs',
+    browserName: 'chrome',
+    platform: 'Windows 7',
+    version: '35'
+  },
+  sl_firefox: {
+    base: 'SauceLabs',
+    browserName: 'firefox',
+    version: '30'
+  },
+  sl_ie_11: {
+    base: 'SauceLabs',
+    browserName: 'internet explorer',
+    platform: 'Windows 8.1',
+    version: '11'
+  }
+};
+
+module.exports = function(config) {
+  config.set({
+      reporters: ['saucelabs', 'spec'],
```

Comment:
There's a lot of duplication with `karma.config.js` here.

Can you DRY this up, perhaps using `object-assign` to merge defaults with overrides (like we do with the webpack configs)?